MAUI/TreeMap/legend.md

```diff
@@ -617,6 +617,147 @@ N> This property will only be applicable when the [ShowLegend](https://help.sync
+    <local:PopulationViewModel />
+    </treemap:SfTreeMap.BindingContext>
+    <treemap:SfTreeMap.Resources>
+        <FlexLayout x:Key="legendLayout"
+                    Padding="10,10,10,10"/>
+    </treemap:SfTreeMap.Resources>
+
+    <treemap:SfTreeMap.LegendSettings>
+        <treemap:TreeMapLegendSettings ShowLegend="True"
+                                        ItemsLayout="{StaticResource legendLayout}"/>
+    </treemap:SfTreeMap.LegendSettings>
+    
+    <treemap:SfTreeMap.LeafItemSettings>
+        <treemap:TreeMapLeafItemSettings LabelPath="Country"/>
+    </treemap:SfTreeMap.LeafItemSettings>
+
+    <treemap:SfTreeMap.LeafItemBrushSettings>
+        <treemap:TreeMapRangeBrushSettings>
+            <treemap:TreeMapRangeBrushSettings.RangeBrushes>
+                <treemap:TreeMapRangeBrush LegendLabel="50M - 1B"
+                                                    From="50000000"
+                                                    To="1000000000" 
+                                                    Brush ="#3F8D71" />
+                <treemap:TreeMapRangeBrush LegendLabel="10M - 50M"
+                                                    From="10000000"
+                                                    To="50000000" 
+                                                    Brush ="#5BA985" />
+                <treemap:TreeMapRangeBrush LegendLabel="0.1M - 10M"
+                                                    From="100000" 
+                                                    To="10000000"  
+                                                    Brush= "#7DC59D" />
+            </treemap:TreeMapRangeBrushSettings.RangeBrushes>
+        </treemap:TreeMapRangeBrushSettings>
+    </treemap:SfTreeMap.LeafItemBrushSettings>
+
+</treemap:SfTreeMap>
+
+{% endhighlight %}
+{% highlight C# hl_lines="2" %}
+
+SfTreeMap treeMap = new SfTreeMap();
+PopulationViewModel viewModel = new PopulationViewModel();
+treeMap.DataSource = viewModel.PopulationDetails;
+treeMap.PrimaryValuePath = "Population";
+treeMap.RangeColorValuePath = "Population";
+treeMap.LegendSettings = new TreeMapLegendSettings() { ShowLegend = true, ItemsLayout = new FlexLayout() { Padding = 10, Background = Colors.LightGreen } };
+treeMap.LeafItemSettings = new TreeMapLeafItemSettings() { LabelPath = "Country" };
+treeMap.LeafItemBrushSettings = new TreeMapRangeBrushSettings()
+{
+    RangeBrushes = new List<TreeMapRangeBrush>()
+    {
+        new TreeMapRangeBrush { LegendLabel = "50M - 1B", From = 50000000, To = 1000000000, Brush = new SolidColorBrush(Color.FromArgb("#3F8D71")) },
+        new TreeMapRangeBrush { LegendLabel = "10M - 50M", From = 10000000, To = 50000000, Brush = new SolidColorBrush(Color.FromArgb("#5BA985")) },
+        new TreeMapRangeBrush { LegendLabel = "0.1M - 10M", From = 100000, To = 10000000, Brush = new SolidColorBrush(Color.FromArgb("#7DC59D")) },
+    }
+};
+ 
+this.Content = treeMap;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Items Template
```

Comment:
Items Template - change as Customize items template

Reply:
updated the changes